src/client/profile-provider.ts

```diff
@@ -181,6 +185,24 @@ export class BoundProfileProvider {
 
     return ok(result.value);
   }
+
+  private mergeParameters(
+    parameters?: Record<string, string>,
+    providerParameters?: Record<string, string>
+  ): Record<string, string> {
+    if (parameters === undefined) {
+      return providerParameters ?? {};
```

Comment:
Why not possibly return undefined? Map interpreter takes parameters as optional parameter

Reply:
you're right. to be honest i didn't write this code, copilot did :grin:

Reply:
oh nice 😄 could it write it correctly if you add undefined as return type too?

Reply:
hard to say, it wrote the return type itself :D